internal/api/provider/oidc_test.go

```diff
@@ -14,25 +18,55 @@ type realIDToken struct {
 	IDToken     string
 	Time        time.Time
 	Email       string
+	Verifier    func(context.Context, *oidc.Config) *oidc.IDTokenVerifier
+}
+
+func googleIDTokenVerifier(ctx context.Context, config *oidc.Config) *oidc.IDTokenVerifier {
+	keyBytes, err := base64.RawURLEncoding.DecodeString("pP-rCe4jkKX6mq8yP1GcBZcxJzmxKWicHHor1S3Q49u6Oe-bQsk5NsK5mdR7Y7liGV9n0ikXSM42dYKQdxbhKA-7--fFon5isJoHr4fIwL2CCwVm5QWlK37q6PiH2_F1M0hRorHfkCb4nI56ZvfygvuOH4LIS82OzIgmsYbeEfwDRpeMSxWKwlpa3pX3GZ6jG7FgzJGBvmBkagpgsa2JZdyU4gEGMOkHdSzi5Ii-6RGfFLhhI1OMxC9P2JaU5yjMN2pikfFIq_dbpm75yNUGpWJNVywtrlNvvJfA74UMN_lVCAaSR0A03BUMg6ljB65gFllpKF224uWBA8tpjngwKQ")
```

Comment:
Would this be the JWKS key? For future reference, where should we look to find a replacement if needed

Reply:
Here https://accounts.google.com/.well-known/openid-configuration then JWKS URL (which may change then), and then one of the keys.

Reply:
Thanks!